build.proj

```diff
@@ -30,6 +31,61 @@
     <RemoveDir Directories="$(BaseOutputPath)" />
   </Target>
 
+  <Target Name="CopyAndDisassembleReferenceOnlyPackages"
+          AfterTargets="Build"
+          Condition="'$(OfflineBuild)' != 'true'">
+
+    <CopyReferenceOnlyPackages
+        PackageCacheDir="$(PackagesDir)"
+        DllDestinationDir="$(ReferencePackagesSourceDir)"
+        IdentifiedPackagesDir="$(ReferencePackagesToDeleteDir)"
+        DestinationDir="$(ReferencePackagesStagingDir)"
+        />
+
+    <!-- Ildasm has trouble writing to paths that are longer than 260 chars. (see https://github.com/dotnet/coreclr/issues/20397)
+         Create a temporary path to write il output which will then be copied back to the reference-packages/source dir.
+    -->
+    <PropertyGroup>
+      <TempPath>$([System.IO.Path]::GetTempPath())</TempPath>
+      <IldasmTempOutputFolderName>$([System.IO.Path]::GetRandomFileName())</IldasmTempOutputFolderName>
+      <IldasmTempOutputPath>$([System.IO.Path]::Combine($(TempPath),$(IldasmTempOutputFolderName)))/</IldasmTempOutputPath>
+    </PropertyGroup>
+    
+    <ItemGroup>
+      <ReferenceOnlyPackages Include="$(ReferencePackagesToDeleteDir)/**/*.nupkg" />
+      <ReferenceOnlyPackageDlls Include="$(ReferencePackagesSourceDir)/**/*.dll" />
+    </ItemGroup>
+
+    <!-- Temporary workaround to exclude dlls that can't be round-tripped. See: https://github.com/dotnet/coreclr/issues/20262
+         They'll end up in the source directory as dlls for now 
+    -->
+    <ItemGroup>
+      <ReferenceOnlyPackageDlls Remove="$(ReferencePackagesSourceDir)/**/System.Runtime.dll;$(ReferencePackagesSourceDir)/**/netstandard.dll" />
```

Comment:
(Nit) Item elements that include multiple files are clearer to me when there's a newline after the `;` or they're just done with multiple elements.